components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -213,41 +221,49 @@ const TrialBenefitsModal = ({
         svgWrapperClassName,
         svgElement,
         bottomLeftMessage,
+        isCloud,
     }: TrialBenefitsModalStepProps) => {
+        // when we are in cloud, ommit the cta to go to the system console, this because the license changes take some time to get applied (see MM-44463)
```

Comment:
If we are adding the Spinner, if we redirect them tot he System Console they will see this correct? Wondering if we still want to keep that button for Cloud.